### PR TITLE
Fix grammar and useless content

### PR DESCRIPTION
**Pull request by angiie**
Change some half-angle punctuation to full-angle and correct some spelling error ,clean up other useless contents.

### CODE REVIEW

**Comment by rainchen** [comment]
@angiie should not use Chinese as commit message

File: website/translated_docs/en-US/cita/advanced-use/contracts/rust.md [hunk 1]
```diff
@@ -268,4 +268,4 @@ $ curl -X POST --data '{"jsonrpc":"2.0","method":"call", "params":[{"to":"0x0000
 {"jsonrpc":"2.0","id":2,"result":"0x0000000000000000000000000000000000000000000000000000000000000011"}
 ```
 
-符合我们编写的智能合约预期。
\ No newline at end of file
+符合我们编写的智能合约预期。
```
Comment (rainchen) [inline website/translated_docs/en-US/cita/advanced-use/contracts/rust.md:271]: should we keep trailing newline? @leeyr338

Comment (angiie) [reply website/translated_docs/en-US/cita/advanced-use/contracts/rust.md:271]: > @angiie should not use Chinese as commit message

Got it!

File: docs/cita/upgrade/protocol-upgrade.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ v0.19 之前的版本链升级 CITA 最新版本需要重新创建链，无法
 
 ### 注意
 
-交易中的 version 字段必须填 0(默认也是 0)。v0.19 并没有校验 version，填非 0 值也可以正常工作。 v0.20 增加了对 version 的校验。因此，如果用户之前发送过 version 不为 0 的交易，升级 v0.20 版本之后，查询这些交易或者新节点同步过程中可能会出现异常情况。可以通过快照方式解决，不影响协议升级操作。
+交易中的 version 字段必须填 0(默认也q是 0)。v0.19 并没有校验 version，填非 0 值也可以正常工作。 v0.20 增加了对 version 的校验。因此，如果用户之前发送过 version 不为 0 的交易，升级 v0.20 版本之后，查询这些交易或者新节点同步过程中可能会出现异常情况。可以通过快照方式解决，不影响协议升级操作。
```
Comment (rainchen) [inline docs/cita/upgrade/protocol-upgrade.md]: `默认也q是`?

**Review by rainchen — DISMISSED** [review]

**Review by jerry-yu — DISMISSED** [review]

**Review by QingYanL — DISMISSED** [review]

**Comment by rainchen** [comment]
@angiie Conflicting files
please update & rebase

**Review by angiie — COMMENTED** [review]
review completed!

**Comment by leeyr338** [comment]
pls rebase the latest master branch.

**Review by leeyr338 — APPROVED** [review]

**Review by nanmuxiusi — APPROVED** [review]

**Pull request merged**